frontend/src/hooks/useDjangoSession.ts

```
@@ -0,0 +1,46 @@
+      setIsSyncing(true)
+
+      syncSession({
+        variables: {
+          accessToken: (session as ExtendedSession).accessToken,
+        },
+      })
+        .then(() => {
+          sessionStorage.setItem(SYNC_STATUS_KEY, 'true')
+        })
+        .catch((err) => {
+          throw new Error('Failed to sync Django session: ' + err.message)
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)